assets/src/edit-story/elements/text/edit.js

```diff
@@ -255,6 +292,36 @@ function TextEdit({
     maybeEnqueueFontStyle(fontFamily);
   }, [fontFamily, maybeEnqueueFontStyle]);
 
+  // If content selected has bold, italic and/or underline, the respective buttons on Design Panel should be activated
+  useEffect(() => {
+    if (editorState) {
+      setProperties({
+        hasBold: editorState.getCurrentInlineStyle().has('BOLD'),
```

Comment:
Actually it works, but not is the best way to go, once I move it this draft to apply these UI behaviors in an implicit way, it will not be necessary anymore. 👍